### DISCUSSION TITLE
Language selection in trending

### DISCUSSION THREAD

**Discussion by niladam** [community/community#3154] [Mobile]
I would love to be able to select the language in the trending section of the application.

**Comment by rnystrom** [comment]
Thanks for the feedback! Can you tell me more about what you want to be able to by seeing trending repositories per language?

**Reply by niladam** [reply]
Sorry but..what are you asking?

If you're trying to ask what I'd want to see it's the same language selection (PHP, HTML, etc) as the GitHub trending pages ?;)

**Reply by rnystrom** [reply]
Yup that's helpful!
